apps/server/src/stores/token-player-map.ts

```diff
@@ -24,11 +28,66 @@ export class TokenPlayerMap {
     return null;
   }
 
-  // update player's socket id when player reconnects
+  /** Update existing player socket id. If player does not exist, throw error.
+   * @param authToken - auth token of the player
+   * @param socketId - socket id of the player
+   */
   updatePlayerSocketId(authToken: AuthToken, socketId: socketId): void {
     const data = this.map.get(authToken);
     if (data) {
       this.map.set(authToken, { ...data, socketId });
+    } else {
+      throw new Error('Player does not exist');
     }
   }
+
+  /** Get player id using socket id.
+   * @param socketId - socket id of the player
+   * @returns player id if player exists, null otherwise
+   */
+  getPlayerIdBySocketId(socketId: socketId): PlayerId | null {
+    for (const [_key, value] of this.map.entries()) {
+      if (value.socketId === socketId) {
+        return value.playerId;
+      }
+    }
+    return null;
+  }
+
+  /** Get socket id using player id.
+   * @param playerId - player id of the player
+   * @returns socket id if player exists, null otherwise
+   */
+  getSocketIdByPlayerId(playerId: PlayerId): socketId | null {
+    for (const [_key, value] of this.map.entries()) {
+      if (value.playerId === playerId) {
+        return value.socketId;
+      }
+    }
+    return null;
+  }
+
+  /** Get auth token using socket id.
+   * @param socketId - socket id of the player
+   * @returns auth token if player exists, null otherwise
+   */
+  getAuthTokenBySocketId(socketId: socketId): AuthToken | null {
+    for (const [key, value] of this.map.entries()) {
+      if (value.socketId === socketId) {
+        return key;
+      }
+    }
+    return null;
+  }
+
+  viewMap(): void {
+    console.log(this.map);
+  }
```

Comment:
This function changed in the newest PRs.
28bc2e9: feat(server): make store state and token player map viewable from the server as a static